draft-ietf-pquip-pqc-engineers.md

```diff
@@ -357,7 +393,7 @@ where pk is public key, sk is secret key, ct is the ciphertext representing an e
                       | Client  | | Server  |
                       +---------+ +---------+
   +----------------------+ |           |
-  | sk, pk = kemKeyGen() |-|           |
+  | pk, sk = kemKeyGen() |-|           |
```

Comment:
Is there a preferred order or is this just matching convention earlier in the document?

Reply:
The output should be (pk, sk) to maintain consistency.